src/zos_console/zowe/zos_console_for_zowe_sdk/console.py

```diff
@@ -43,3 +43,24 @@ def issue_command(self, command, console=None):
         custom_args["json"] = request_body
         response_json = self.request_handler.perform_request("PUT", custom_args)
         return response_json
+
+    def get_response(self, response_key, console=None):
+        """
+        Collect outstanding synchronous z/OS Console response messages.
+        
+        Parameters
+        ----------
+        response_key
+            The command response key from the Issue Command request.
+        console
+            The console that should be used to get the command response.
+        Returns
+        -------
+        json
+            A JSON containing the response to the command
+        """
+        custom_args = self._create_custom_request_arguments()
+        request_url = "{}/solmsgs/{}".format(console or "defcn", response_key)
```

Comment:
should I also create test for this?

Reply:
Yes, please add unit and integration tests 🙂 They can be modelled after tests that you've created for previous PRs.